src/styles/blocks/star.scss

```diff
@@ -0,0 +1,19 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-image: url(/images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  &__star:not(:last-child) {
+    margin-right: 4px;
+  }
+
+  &__star:nth-child(-n+4) {
+    background-image: url(/images/star-active.svg);
+  }
```

Comment:
change the styles
![image](https://github.com/mate-academy/layout_catalog/assets/22824431/68cbd4f1-7081-4e20-acce-
b94021eba509)